Smith_Ben/Assignments/Dojo_Survey copy/templates/index.html

```diff
@@ -0,0 +1,48 @@
+<html>
+    <head>
+       <title>Dojo Suvey</title>
+    </head>
+    <body>
+      <h1>Give me yo info PUNK!</h1>
+       {% with messages = get_flashed_messages() %}
+    {% if messages %}
+      {% for message in messages %}
+        <p>{{message}}</p>
+      {% endfor %}
+    {% endif %}
+  {% endwith %}
+      <form action='/users' method='post'>
+          Name: <input type='text' name="{{'name'}}">
```

Comment:
no need to use python embedding in name/value attributes.  can just be <input type='text' name='name'>